### PR TITLE
Open checkstyle, change checkstyle severity from info to warning

### PR DESCRIPTION
**Pull request by ruanwenjun**
## Purpose of the pull request

Right now, the checkstyle is unused, since we set severity level to info,  this means when someone break the checkstyle rule, the will be a info log, the CI/checkstyle task will still pass.

todo: We need to config a checkstyle task on ci.

## Brief change log
Change checkstyle severity to warning.

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by kezhenxu94 — DISMISSED** [review]

File: .github/workflows/api-test.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ jobs:
           ./mvnw -B clean install \
           -Dmaven.test.skip \
           -Dmaven.javadoc.skip \
-          -Dmaven.checkstyle.skip \
+          -Dcheckstyle.skip=true \
```
Comment (SbloodyS) [inline .github/workflows/api-test.yml:68]: Should we skip checkstyle in ```e2e.yaml```?

Comment (ruanwenjun) [reply .github/workflows/api-test.yml:68]: I think we can skip, we need to add a single checkstyle task, this need to wait we solve the all exist code format errors.

Comment (SbloodyS) [reply .github/workflows/api-test.yml:68]: > I think we can skip, we need to add a single checkstyle task, this need to wait we solve the all exist code format errors.

That would be great.

**Review by SbloodyS — APPROVED** [review]
+1

**Pull request merged**